plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1586,6 +1586,9 @@ preferences.blockly.blockRenderer=Blocks renderer
 preferences.blockly.blockRenderer.description=The renderer (display style) of the blocks in the Blockly editor
 preferences.blockly.useSmartSort=Use smart procedure block sorting
 preferences.blockly.useSmartSort.description=When enabled, procedure blocks will be smart-sorted, otherwise they will be sorted alphabetically
+preferences.blockly.expandCategories=Expand categories
+preferences.blockly.expandCategories.description=This defines how nested categories should be expanded when opening a Blockly editor. Default will use the category's setting.<br>\
```

Comment:
' will not work as it is not escaped correctly

Reply:
Oh... it was too small in the UI, so I didn't notice 😬